agenta-web/src/lib/helpers/evaluate.ts

```diff
@@ -26,17 +38,29 @@ export const exportABTestingEvaluationData = (evaluation: Evaluation, rows: Gene
     downloadCsv(csvData, filename)
 }
 
-export const exportSingleModelEvaluationData = (evaluation: Evaluation, rows: GenericObject[]) => {
+export const exportSingleModelEvaluationData = (
+    evaluation: Evaluation,
+    scenarios: EvaluationScenario[],
+    rows: GenericObject[],
+) => {
     const exportRow = rows.map((data, ix) => {
+        const inputColumns = data.inputs.reduce(
```

Comment:
```suggestion
        const inputColumns = evaluation.testset.testsetChatColumn
            ? {Input: evaluation.testset.csvdata[ix]?.[evaluation.testset.testsetChatColumn]}
            : data.inputs.reduce(
```

To cater to chat apps